src/pages/Dummy/index.js

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import { ROUTES } from '../../routerConfig';
+import LolButton from '../../components/lolButton';
+import InputField from '../../components/inputField';
+import DialogButton from '../../components/dialogField';
+import { useState } from 'react';
+import DialogField from '../../components/dialogField';
+import Snackbar from '../../components/simpleSnackbar';
 function Dummy(){
+    const [open, setOpen] = useState(false);
+    const dialogTitle = "Dummy Title";
+    const dialogText = "Dummy Text";
+    const snackbarButtonType = "error";
```

Comment:
snackbarButtonType -> 
export const SNACKBAR_TYPES = {
     ERROR: "error"
}

and move it before function Dummy()